src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -440,7 +440,7 @@
             </trans-unit>
             <trans-unit id="113">
                 <source>This URL does not contain a TLD.</source>
-                <target state="needs-review-translation">Esta URL no contiene un dominio de nivel superior (TLD).</target>
+                <target>Esta URL no contiene una extensión de dominio (TLD).</target>
```

Comment:
Thanks Ezequiel for reviewing this translation. However, I'd just simplify it as follows:

`Esta URL no contiene una extensión de dominio.`

My guess is that `TLD` is an acronym that nobody outside of tech understands.

Reply:
We kept TLD in other languages in case it rigns a more precise bell in the mind of reader (also might help googling)
I think we should keep it.

Reply:
I like using the term "domain extension". The original sentence doesn't contain it (nor do the other validated languages). Let's maybe improve the original sentence and then say the same thing in all languages?

"This URL does not contain a domain extension (TLD)."